src/index.jsx

```diff
@@ -0,0 +1,17 @@
+import '@babel/polyfill';
+import React from 'react';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+import { store } from './store';
+import App from './components/App';
+import 'normalize.css/normalize.css';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
👀

Reply:
I got this from a tutorial I watched. Is there a better way for me to use this package?